examples/clean-state/README.md

```diff
@@ -0,0 +1,16 @@
+# Clean state contract in JavaScript
+
+This is an equivalent JavaScript implementation of the clean state example. Every user can send and receive a non-fungible token.
```

Comment:
```suggestion
This is an equivalent JavaScript implementation of the clean state example. 
```

Reply:
Yeah I keep forgetting to update all the stuff, thanks for keeping your eyes open, haha